src/utils/set_proof_api_url.ts

```diff
@@ -7,4 +7,14 @@ export const setProofApi = (url: string) => {
     }
     url += 'api/v1/';
     service.network = new NetworkService(url);
-};
\ No newline at end of file
+};
+
+export const setHermezProofApi = (url: string, override = true) => {
```

Comment:
what is need of additional param, override ?

I think  if user calls it which means he wants to override .

Reply:
so in hermez client init function we set this hermezProofApi by default and we don't want that part of the code to over write the user API that is why in that function we are using the value 'false'